src/main/java/nc/students/ayaz/controllers/VideosController.java

```diff
@@ -0,0 +1,38 @@
+package nc.students.ayaz.controllers;
+
+
+import nc.students.ayaz.model.User;
+import nc.students.ayaz.model.Video;
+import nc.students.ayaz.model.exceptions.NoSuchVideoException;
+import nc.students.ayaz.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/users/{nickname}/videos")
+public class VideosController {
+
+    @Autowired
+    private UserRepository repository;
+
+    @PostMapping("/{videoName}")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createVideo(@PathVariable String nickname, @PathVariable String videoName) throws Exception {
```

Comment:
&#x1F34E; **Issue**
Why do you throw Exception, while handle only specific type of Exception (NoSuchVideoException).
E.g. if I'll try to send a GET request on "/users/nonexistent_user/videos/whatever_video" I will get 500 instead of 404.